docs/DeveloperGuide.md

```diff
@@ -449,13 +466,64 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 3.
 
+* 3c. User enters an invalid contact information
+   * 3c1. System displays that the index is invalid and to try again.
```

Comment:
Should "index" be changed to "contact information"?

Reply:
Changed to "contact information", good catch!